src/superphot_plus/fit_numpyro.py

```diff
@@ -550,7 +553,7 @@ def main_loop_directory(test_filenames, output_dir=FITS_DIR):
     lcs = []
     for filename in test_filenames:
         lc = Lightcurve.from_file(filename)
-        lc.pad_bands(["g", "r"], PAD_SIZE)
+        lc.pad_bands(["r", "g"], PAD_SIZE)
```

Comment:
This switches the order of the bands. I don't think it matters due to all of the other changes in the functions, but I wanted to highlight it in case this was not intentional.

Reply:
Yep! Did it to match the ztf YAML band order